pkg/northbound/gnmi/set.go

```diff
@@ -280,6 +267,27 @@ func listenForDeviceResponse(changes mapNetworkChanges, target string, name stor
 	}
 }
 
+func doRollback(changes mapNetworkChanges, mgr *manager.Manager, target string,
+	name store.ConfigName, errResp error) error {
+	rolledbackIDs := make([]string, 0)
+	for configName := range changes {
+		changeID, err := mgr.RollbackTargetConfig(string(configName))
```

Comment:
maybe RollbackTargetConfig() should take a **ConfigName** rather than string

Reply:
yeah, that's part fo the configName/string refactor issue I opened. don't want to do it here